src/vs/base/browser/ui/selectBox/selectBoxCustom.ts

```diff
@@ -520,12 +520,7 @@ export class SelectBoxList extends Disposable implements ISelectBoxDelegate, ILi
 		return {
 			dispose: () => {
 				// contextView will dispose itself if moving from one View to another
-				try {
-					container.removeChild(this.selectDropDownContainer); // remove to take out the CSS rules we add
-				}
-				catch (error) {
-					// Ignore, removed already by change of focus
-				}
+				this.selectDropDownContainer.remove(); // remove to take out the CSS rules we add
```

Comment:
Could this throw? why was it try/catch before?

Reply:
No. `parent.removeChild(child)` throws if `child` is not a child of `parent`. `child.remove()` never throws. Aside from not needing a reference to the parent, that’s the only difference.